shared_model/interfaces/transaction_responses/tx_response.hpp

```diff
@@ -65,7 +65,7 @@ namespace shared_model {
       /**
        * @return attached concrete tx response
        */
-      virtual const ResponseVariantType &get() const = 0;
+      virtual ResponseVariantType get() const = 0;
```

Comment:
why remove const here?

Reply:
https://isocpp.org/blog/2013/06/quick-q-should-i-return-a-const-value-stackoverflow

> The answer is now a definite “No” because it prevents move from returned values.